app/models/ability.rb

```diff
@@ -20,7 +20,7 @@ def initialize(user)
     end
 
     can :read, Document do |d|
-      d.public? and ( d.organization.blank? or d.organization.participants.include?(user.participant) )
+      d.public? or ( d.organization.participants.include?(user.participant) )
```

Comment:
Crashes when there is a document that doesn't have an organization as specified in issue #93. (d.organization is nil and thus everything after is a null pointer exception).

Please modify this to use an if statement that checks whether it the doc has an org attached to it.